internal/metrics/metrics.go

```diff
@@ -89,6 +92,13 @@ const (
 // to regenerate the metrics documentation.
 func NewMetrics(registry *prometheus.Registry) *Metrics {
 	m := Metrics{
+		buildInfoGauge: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: BuildInfoGauge,
+				Help: "Build information for Contour set at compile time. branch - the branch contour was built from, revision - the head git sha at the time contour was built, version - the tagged git version at the time Contour was built.",
```

Comment:
I tried a few different variations and trimmed the help down to this:
```
Build information for Contour. The labels are the branch and git SHA that Contour was built from, and the Contour version.
```

What do you think?

Reply:
I am all for a shorter variation. Another potential one.

> Build information for Contour. Labels include the branch and git SHA that Contour was built from, and the Contour version.

Reply:
Yep, that sounds good 👍